Prolangle/Services/SnippetRevealer.cs

```diff
@@ -35,6 +33,8 @@ public string RevealMore()
 		concealedCode = concealedCode[..revealStart] + sourceCode.Substring(revealStart, revealEnd - revealStart) +
 		                concealedCode[revealEnd..];
 
+		RevealedProgress += 0.05;
```

Comment:
Note that this is a misnomer now. It's "progress to reveal next time".

Reply:
That's fine. Maybe add a comment so we still know why it's at that position a year from now.

Reply:
Yeah.